packages/autorest.python/autorest/codegen/models/model_type.py

```diff
@@ -229,10 +246,26 @@ def discriminator(self) -> Optional[Property]:
         except StopIteration:
             return None
 
+    @property
+    def _discriminator_to_me(self) -> Optional[Property]:
```

Comment:
Is this finding the discriminator for a given model? I would rename this to be `discriminator_property`. If this property will be used by code outside of this file, it should be a public property

Reply:
yes, for finding discriminator property pointing to the `self` model. Renamed and made it public.